doc/rados/operations/placement-groups.rst

```diff
@@ -559,100 +559,103 @@ resources.
-designed your Ceph cluster to maximize `data durability`_,
-`object distribution`_ and minimize `resource usage`_.
+Next, check whether the resulting baseline value comports with the way you
+designed your Ceph cluster to maximize `data durability`_ and `object
+distribution`_ and to minimize `resource usage`_.
 
-The result should always be **rounded up to the nearest power of two**.
+This value should always be **rounded up to the nearest power of two**.
 
-Only a power of two will evenly balance the number of objects among
-placement groups. Other values will result in an uneven distribution of
-data across your OSDs. Their use should be limited to incrementally
-stepping from one power of two to another.
+Only a power of two will evenly balance the number of RADOS objects across PGs.
+Other values will result in an uneven distribution of data across your OSDs. Do
+not increase the number of PGs unless it is possible to change the number of
+PGs to the nearest power of two.
 
-As an example, for a cluster with 200 OSDs and a pool size of 3
-replicas, you would estimate your number of PGs as follows
+For example, if you have a cluster with 200 OSDs and a pool size of 3 replicas,
+estimate the number of PGs as follows:
 
-  :math:`\frac{200 \times 100}{3} = 6667`. Nearest power of 2: 8192
+  :math:`\frac{200 \times 100}{3} = 6667`. Rounded up to the nearest power of 2: 8192.
 
-When using multiple data pools for storing objects, you need to ensure
-that you balance the number of placement groups per pool with the
-number of placement groups per OSD so that you arrive at a reasonable
-total number of placement groups that provides reasonably low variance
-per OSD without taxing system resources or making the peering process
-too slow.
+When using multiple data pools to store objects, make sure that you balance the
+number of PGs per pool against the number of PGs per OSD so that you arrive at
+a reasonable total number of PGs. It is important to find a number that
+provides reasonably low variance per OSD without taxing system resources or
+making the peering process too slow.
 
-For instance a cluster of 10 pools each with 512 placement groups on
-ten OSDs is a total of 5,120 placement groups spread over ten OSDs,
-that is 512 placement groups per OSD. That does not use too many
-resources. However, if 1,000 pools were created with 512 placement
-groups each, the OSDs will handle ~50,000 placement groups each and it
-would require significantly more resources and time for peering.
+For example, suppose you have a cluster of 10 pools, each with 512 PGs on 10
+OSDs. That amounts to 5,120 PGs distributed across 10 OSDs, or 512 PGs per OSD.
+This cluster will not use too many resources. However, in a cluster of 1,000
+pools, each with 512 PGs on 10 OSDs, the OSDs will have to handle ~50,000 PGs
+each. This cluster will require significantly more resources and significantly
+more time for peering.
```

Comment:
We've seen people actually do this, like one pool per user / customer.  When you have multiple device classes it can get more complicated, but if we went there in this document we'd probably frighten more people than we'd help.

Reply:
I suspect that by this time next year, I will have clear a space for a book called something like "Weird Tales of Ceph", and we will be able to get into the details of convoluted, eldritch deployments.

Reply:
When we wrap around to C again in 2024 we must ensure that the release is named Cthulhu as Cuttlefish should have been.

Reply:
When I saw "2034", my first thought was "I'll be dead by then anyway", and then I realized that 2034 was only eleven years in the future.

Reply:
Let's wait for the 2024 election results before we make any assumptions